symphony/orchestration/simbricks/orchestration/simulation/host.py

```diff
@@ -262,18 +284,13 @@ async def prepare(self, inst: inst_base.Instantiation) -> None:
 
         d = []
         for disk in full_sys_hosts[0].disks:
-            format = (
-                "qcow2"
-                if not isinstance(disk, sys_host.LinuxConfigDiskImage)
-                else "raw"
-            )
-            copy_path = await disk.make_qcow_copy(
-                inst=inst,
-                format=format,
-                sim=self
-            )
-            assert copy_path is not None
-            d.append((copy_path, format))
+            if not isinstance(disk, disk_images.LinuxConfigDiskImage):
```

Comment:
Erm.. That just feels wrong. ;-) Should we not just check if qcow2 is in the list of available formats (it is not for the config disk images...) and then use that as the if? Or if it's about the "one-off" where a copy is unnecessary, maybe we want to add a flag or something for that? Seems a bit silly if the user has to go in and add an exception here in the base class...